client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1,10 +1,7 @@
 /**
  * External dependencies
- *
- * @format
```

Comment:
AFAIK, this should be left as-is - it's used to tag files that are auto-formatted using `prettier` in our [pre-commit hook](https://github.com/Automattic/wp-calypso/blob/master/bin/pre-commit-hook.js).